CHANGELOG.md

```diff
@@ -6,6 +6,80 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.9.0] Io - 2022-02-22 [:warning:]
+Injectable wire encoding and a `protobuf` wire encoder.
+
+### Added
+
+- :sparkles: **Injectable wire encoding [#233]:** The encoding used for messages that are sent across the network is now injectable.
+  - Add binary marshalling: [#272], [#284], [#298].
+  - Add serializer interface: [#297], [#325].
+  - Changed generation of `PropoposalID`: [#300], [#307].
+  - Rename message types [#305].
+  - Extend and refactor message serialization tests: [#316], [#317].
+- :sparkles: **Protobuf wire encoder [#311]:** A wire encoder based on `protobuf` has been added [#318].
+- Minor additions: Introduce `Asset.Equal` [#279], export proposer index [#313].
+
+### Changed
+
+- :boom: Rename `Address.Equals` to `Address.Equal` [#264].
+- :boom: Revise `Channel.Update` and `Channel.ForceUpdate`: [#289], [#291], [#306].
+- :truck: `pkg/io` migration: [#256], [#271], [#285], [#287].
+- :memo: Updated README to include references to existing backends [#314].
+- :children_crossing: Minor usability improvements: [#268], [#278], [#315].
+- :construction_worker: CI updates: [#260], [#274], [#276], [#277].
+- :arrow_up: Use `LatestSigner` in Ethereum contract backend [#322].
```

Comment:
```suggestion
- :arrow_up: Use `LatestSigner` in Ethereum contract backend and don't set `GasLimit` by default, so that [EIP1559](https://eips.ethereum.org/EIPS/eip-1559) TXs are sent [#322].
```